### DISCUSSION TITLE
How should I bind fallback with IsVisible property?

### DISCUSSION THREAD

**Discussion by kyuranger** [AvaloniaUI/Avalonia#15263] [Q&A] [answered]
![image](https://github.com/AvaloniaUI/Avalonia/assets/30720036/def4a601-48e1-4d25-b6ed-3deeed2b1a63)
Here is my code:
```
<Button Background="{StaticResource NormalBackground}" HorizontalAlignment="Left" Padding="20,10,20,10" VerticalAlignment="Center" Grid.Column="1" Grid.Row="1" Margin="15,0,0,0">
				<Grid ColumnDefinitions="auto,auto">
								<PathIcon Classes="Loading" Height="15" Width="15">
									<PathIcon.Styles>
										<Style Selector="PathIcon">
											<Style.Animations>
												<Animation Duration="0:0:1" IterationCount="INFINITE">
													<KeyFrame Cue="0%">
														<Setter Property="RotateTransform.Angle" Value="0"/>
													</KeyFrame>
													<KeyFrame Cue="100%">
														<Setter Property="RotateTransform.Angle" Value="360"/>
													</KeyFrame>
												</Animation>
											</Style.Animations>
										</Style>
									</PathIcon.Styles>
								</PathIcon>
								<TextBlock Margin="10,0,0,0" Grid.Column="1" Foreground="{StaticResource SecondaryForeground}" Text="Refresh"></TextBlock>
				</Grid>
</Button>
```
The effect I want to do is very simple. 

By default, the PathIcon in the Button is invisible. 

When the Refresh button is clicked, the PathIcon inside the Button becomes visible, and the IsEnabled of the Button itself becomes false. Button will bind a method to read data from the remote server. 

When the method execution ends, the PathIcon in the Button becomes invisible again, and the IsEnabled of the Button itself becomes true. 

How can I achieve this?

**Comment by timunie** [comment, marked as answer]
IsVisible is a bool that can be bound to any other bool property of your ViewModel

**Reply by kyuranger** [reply]
Well, it seems that this is the only way to use it.
